### PR TITLE
[timeline] fix unit bug

### PR DESCRIPTION
**Pull request by chejennifer**

### CODE REVIEW

File: static/js/tools/timeline/chart.tsx [hunk 1]
```diff
@@ -333,8 +333,12 @@ class Chart extends Component<ChartPropsType, ChartStateType> {
     for (const statVar in statData.data) {
       for (const place in statData.data[statVar]) {
         const series = statData.data[statVar][place];
-        if (series && series.facet && statData.facets[series.facet].unit) {
-          units.add(statData.facets[series.facet].unit);
+        if (!series || !series.facet || !statData.facets[series.facet]) {
```
Comment (shifucun) [inline static/js/tools/timeline/chart.tsx:336]: what is the actual statData looks like in this case?

**Review by shifucun — APPROVED** [review]

File: static/js/tools/timeline/chart.tsx (hunk 1 above)
Comment (chejennifer) [reply static/js/tools/timeline/chart.tsx:336]: looked into it and there was a problem with updating the statData when there was a mix of modelStatVars and non modelStatVars where only the facet information for modelStatVars were included. Updated this PR with a fix for this.

**Review by shifucun — APPROVED** [review]
thanks!

**Comment by chejennifer** [comment]
thanks for the review!

**Pull request merged**